src/services/iiif-parser.js

```diff
@@ -122,3 +121,41 @@ export function getMediaFragment(uri) {
     return undefined;
   }
 }
+
+export function getCanvas(uri) {
+  if (uri !== undefined) {
+    return uri.split('#t=')[0];
+  }
+}
+
+export function getSources(choiceItems) {
+  const sources = choiceItems.map(item => {
+    return {
+      src: item.id,
+      // TODO: make type more generic, possibly use mime-db
+      format: item.getFormat().value
+    };
+  });
+  return sources;
+}
+
+export function getMediaType(choiceItems) {
+  let allTypes = choiceItems.map(item => item.getType().value);
+  let uniqueTypes = allTypes.filter((t, index) => {
+    return allTypes.indexOf(t) === index;
+  });
+  if (uniqueTypes.length === 1) {
+    return uniqueTypes[0];
+  }
+  // Default type if there are different types
+  return 'audio';
```

Comment:
Should we default to video instead of audio?

Reply:
Sounds good :+1: